ui/imports/utils/Global.qml

```diff
@@ -14,8 +14,9 @@ QtObject {
     property var mainModuleInst
     property var toastMessage
     property bool profilePopupOpened: false
-    //Not refactored yet
-    property bool networkGuarded: false //profileModel.network.current === Constants.networkMainnet || (profileModel.network.current === Constants.networkRopsten && localAccountSensitiveSettings.stickersEnsRopsten)
+    property string currentNetworkId: ""
+    property bool networkGuarded: root.currentNetworkId === Constants.networkMainnet ||
+        (root.currentNetworkId === Constants.networkRopsten && localAccountSensitiveSettings.stickersEnsRopsten)
```

Comment:
This property is [part of Global](https://github.com/status-im/status-desktop/blob/a93ebe3366f1684336993bd091945ad224bec895/ui/imports/utils/Global.qml#L18), would mind using that one instead?